pkgs/by-name/op/open-adventure/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  python312,
+  python312Packages,
```

```suggestion
  python3Packages,
```
Are you sure this doesn't work with an unpinned Python?